rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -940,21 +990,10 @@ MasterGraph::copy_output(unsigned char *out_ptr, size_t out_size_in_bytes)
 void MasterGraph::output_routine()
 {
     INFO("Output routine started with "+TOSTR(_remaining_count) + " to load");
```

Comment:
Revert the Sequence reader changes.

Reply:
As discussed in the morning, this change is needed for some of the changes introduced in output_routine_video